AutoGraph/Utilities.swift

```diff
@@ -61,3 +61,5 @@ extension DataResponse {
         return json
     }
 }
+
+extension String: Error {}
```

Comment:
I'm kind of opposed to this. This could break library clients that do the same thing (since you can't conform twice to the same protocol) and fundamentally not all strings are errors. In our use case you can literally create a new type right there with an error conformance, would rather we do that.